examples/recommender_system/recommender_system_env.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict, Tuple
+import gym
+import gym.spaces
+import numpy as np
+
+
+gym.envs.registration.register(
+    id='RecommenderSystemEnv-v1',
+    entry_point='examples.recommender_system.recommender_system_env:RecommenderSystemEnv'
+)
+
+
+class RecommenderSystemEnv(gym.Env):
+    
+    def __init__(self, preferences: Dict) -> None:
+        super().__init__()
+        self.content_tags = list(preferences.keys())
+        self.preferences = preferences  # preferences as probability of enjoying the content
+        self.action_space = gym.spaces.Discrete(len(self.content_tags))
+        self.observation_space = gym.spaces.Space()
+    
+    def reset(self, seed: int = None) -> Tuple[gym.spaces.Dict, Dict]:
```

Comment:
Dlaczego zwracamy `gym.spaces.Dict`, skoro `observation_space` to pusty `Space`?

Reply:
Racja, mój błąd